content/1_General/Contest_Strategy.mdx

```diff
@@ -27,22 +27,23 @@ Historically, the first platinum problem has never been the hardest (though you
 
 ### Time allocation
 
-It's the worst feeling in the world to sink a couple hours into a problem before realizing that a different problem would've been easy pickings; we can easily avoid this situation with proper time allocation. You have to use your own judgement in deciding what is likely solvable and what should be quit. Generally, don't spend "too long" on one problem, and stay away from problems that look like they test something you don't know well.
+It's the worst feeling in the world to sink a couple hours into a problem before realizing that a different problem would've been easy pickings; we can easily avoid this situation with proper time allocation. You have to use your own judgement in deciding what is likely solvable and when you should quit. Generally, don't spend "too long" on one problem, and stay away from problems that look like they test something you don't know well.
 
 ### When you see a solution
 
 If you immediately get the solution to a problem while reading it, it might be a good idea to code it up quickly and get it out of the way.
 
 It also might be a good idea to make some progress on other problems before coding the one you know how to do. When you come back and start coding, you can make sure your solution still makes sense and then you have ideas from the other problems bouncing around in your head while you're coding.
 
-### When there are partials
+### When there are partial accepted answers
```

Comment:
Is "partials" better or "partial accepted answers"? @bqi343

Reply:
I don't think "partial accepted answers" makes sense. "when there is partial scoring" or "when there are subtasks" maybe

Reply:
Oh, I see now, that makes more sense. Thank you.